state/state.go

```diff
@@ -272,7 +273,8 @@ func (st *state) ProposeBlock(round int) (*block.Block, error) {
 
 	stateHash := st.stateHash()
 	timestamp := st.proposeNextBlockTime()
-	newSortitionSeed := st.lastInfo.SortitionSeed().Generate(st.signer)
+	seed := st.lastInfo.SortitionSeed()
+	newSortitionSeed := seed.Generate(st.signer)
 
```

Comment:
I don't see `seed` being used after here And I don't think other than consuming memory of seed variable it has other impact
here as well
https://github.com/zarbchain/zarb-go/pull/229/files#diff-ad95c17c31b539a3e385a23d86bbecf4d8a925ba6679aec2e66b5202ba4c4f81R350-R351

Reply:
It should be, we passed the linting pipeline

Reply:
linter checks for unused variable while this `seed` variable is used in line 277 so the linter won't mind

Reply:
The `Generate` method use pointer to seed. This is why we define `seed` here